projects/storefrontlib/src/cms-components/misc/qualtrics/qualtrics.component.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { WindowRef } from '@spartacus/core';
+import { Observable, of } from 'rxjs';
+import { QualtricsLoaderService } from './qualtrics-loader.service';
+import { QualtricsComponent } from './qualtrics.component';
+
+class MockQualtricsLoaderService {
+  load(): Observable<boolean> {
+    return of(true);
+  }
+}
+
+export class MockQualtricsConfig {
+  qualtrics: {};
+}
+
+describe('QualtricsComponent', () => {
+  let component: QualtricsComponent;
+  let fixture: ComponentFixture<QualtricsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [QualtricsComponent],
+      providers: [
+        WindowRef,
```

Comment:
Is this `WindowRef` needed?

Reply:
Nope fixed, this was a test created from the early stages before